src/components/paradox/results/personality/career/inline-jobs/job.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from "prop-types";
+import {faLocationDot} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
+import DangerousHTML from "lib/helpers/dangerous-html";
+import style from "./style.scss";
+
+function InlineJob({job, jobSource, jobSourceURL, translate}) {
+  if(!job || Object.keys(job).length === 0) {
```

Comment:
I think we could just have this check if `!job`

Reply:
That way it's not confusing when we think we have a job but don't

Reply:
This check is there to render the component when an empty array is passed in. Without it, it will render an empty component instead of "No Jobs Found".

Reply:
Actually Im now passing in null if jobs is an empty array. OK check removed!